docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
 * [Tessera](../HowTo/GetStarted/Install.md#as-release-binaries).
 * [IBFT network as configured in IBFT tutorial](Create-IBFT-Network.md).
 
-### Steps
-
-1. Generate new keys using `java -jar /path-to-tessera/tessera.jar -keygen -filename new-node-1`
-
-    ```bash
-    mkdir new-node-1t
-    cd new-node-1t
-    java -jar ../tessera.jar -keygen -filename new-node-1
-    ```
-
-    Result:
-
-    ```text
-    Enter a password if you want to lock the private key or leave blank
-    ```
-
-    Enter a password.
-
-    ```text
-    Please re-enter the password (or lack of) to confirm
-    ```
-
-    Enter it again.
-
-    Result:
-
-    ```text
-    10:32:51.256 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generating new keypair...
-    10:32:51.279 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generated public key PublicKey[pnesVeDgs805ZPbnulzC5wokDzpdN7CeYKVUBXup/W4=] and private key REDACTED
-    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved public key to /Users/username/fromscratch/new-node-1t/new-node-1.pub
-    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved private key to /Users/username/fromscratch/new-node-1t/new-node-1.key
-    ```
-
-1. Create a `new-node-1t/config.json` configuration file with newly generated keys referenced:
-
-    ```json
-    {
-       "useWhiteList": false,
-       "jdbc": {
-           "username": "sa",
-           "password": "",
-           "url": "jdbc:h2:/yourpath/new-node-1t/db1;MODE=Oracle;TRACE_LEVEL_SYSTEM_OUT=0",
-           "autoCreateTables": true
+## Steps
+
+Listed on the right-hand side of the page are the steps to create a private network using IBFT
+with 5 GoQuorum nodes and 2 Tessera nodes.
+
+### 1. Create directories
+
+Create directories for the 2 Tessera nodes in the `IBFT-Network` directory previously created.
+
+```bash
+IBFT-Network/
+├── Node-0
+│   ├── data
+├── Tessera-0
+├── Node-1
+│   ├── data
+├── Tessera-1
+├── Node-2
+│   ├── data
+├── Node-3
+│   ├── data
+├── Node-4
+│   ├── data
+```
+
+### 2. Generate Tessera keys
+
+In the `Tessera-0` directory, generate keys.
+
+```bash
+java -jar /<path-to-tessera>/tessera.jar -keygen -filename tessera0
+```
+
+Press enter both times you are prompted for a password.
+
+The private and public key are created in files called `tessera0.key` and `tessera0.pub`.
+
+!!! caution
+    In a production environment, ensure keys are secured appropriately.
+
+### 3. Create configuration file
+
+In the `Tessera-0` directory, create a configuration file called `config.json`. Copy and paste the
+the configuration below into the file.
+
+```json
+{
+   "useWhiteList": false,
+   "jdbc": {
+       "username": "sa",
+       "password": "",
+       "url": "jdbc:h2:/<path to IBFT-network>/IBFT-network/Tessera-0/db1;MODE=Oracle;TRACE_LEVEL_SYSTEM_OUT=0",
```

Comment:
Ask for the `<path to IBFT-network>` placeholder to be updated in the file

Reply:
Fixed